src/CFNetwork/CFHTTPMessage.cs

```diff
@@ -80,16 +80,20 @@ static IntPtr GetVersion (Version version)
 				return _HTTPVersion1_1.Handle;
```

Comment:
I wonder if `null` meant _latest_ possible. Seems weird to default to 1.1...

Reply:
I vote for not touching it. Http 1.0 is ooooold